### PR TITLE
feat: [PAGOPA-65] Delete payment type

### PR DESCRIPTION
**Pull request by aomegax**
- depends on https://github.com/pagopa/pagopa-afm-marketplace-be/pull/52
- depends on https://github.com/pagopa/pagopa-infra/pull/566

#### List of Changes
- added call to afm marketplace to check if payment type is used

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/resources/application-azure-t.properties [hunk 1]
```diff
@@ -46,3 +46,7 @@ xsd.cdi=${XSD_CDI}
 # Zip file constraint
 zip.entries=10
 zip.size=5000000
+
```
Comment (jacopocarlini) [inline src/main/resources/application-azure-t.properties:49]: I suggest deleting this file. it is no longer useful

File: pom.xml [hunk 2]
```diff
@@ -60,11 +61,11 @@
             <version>${spring-version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>org.springframework.retry</groupId>
-            <artifactId>spring-retry</artifactId>
-            <version>${spring-retry-version}</version>
-        </dependency>
+<!--        <dependency>-->
```
Comment (jacopocarlini) [inline pom.xml:64]: delete the dependency

**Review by jacopocarlini — COMMENTED** [review]
ok, just small comments

File: src/main/resources/application-azure-t.properties (hunk 1 above)
Comment (aomegax) [reply src/main/resources/application-azure-t.properties]: in the next future I would like unify `application.properties`

**Review by jacopocarlini — APPROVED** [review]
🎖️ Good job

**Pull request merged**